features/fields.feature

```diff
@@ -0,0 +1,228 @@
+#language: nl
+
+@post-assert
+Functionaliteit: Fields
+  Als consumer van de BRP API
+  Wil ik kunnen aangeven welke gegevens van de gevonden personen ik terug wil krijgen
+  Zodat ik alleen de gegevens terug krijg waarvoor ik doelbinding heb
+
+  Achtergrond:
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                             | waarde    |
+    | anummer                          | 123456789 |
+    | burgerservicenummer              | 999995078 |
+    | geslachtsaanduiding (04.10)      | M         |
+    | geslachtsaanduiding.omschrijving | man       |
+    En de persoon heeft de volgende 'naam' gegevens
+    | naam          | waarde   |
+    | geslachtsnaam | Groenen  |
+    | voornamen     | Franklin |
+    En de persoon heeft een partner met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999992935 |
+
+  Scenario: Gevraagde veld is een 'root' gegeven van de persoon
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 999995078                       |
+    | fields              | burgerservicenummer             |
+    Dan heeft de persoon met burgerservicenummer '999995078' de volgende gegevens
```

Comment:
moet hier niet het woord "alleen" worden toegevoegd? De Dan stap moet controleren dat er geen andere velden dan wat gevraagd is wordt geleverd. Zelfde bij alle volgende stappen.

Reply:
is toegevoegd